### ISSUE TITLE
How do you find out the ID for a feature?

### ISSUE THREAD

**Issue by o-t-w** [web-platform-dx/baseline-status#49]
Is there a list of all the possible ids for all the different features for people to search through? It's not always obvious.

**Comment by conshus** [comment]
I'm sure someone will reply with a better answer, but I think you can search for the feature at https://webstatus.dev/ and the slug of the page will be what you need to pass into the Web Component. Hope that helps.

**Comment by captainbrosset** [comment]
@o-t-w You could also use the [explorer website](https://web-platform-dx.github.io/web-features-explorer/features/) to see all features on a single page. I recently added a search tool too. But I'm not sure what your use case is, so not sure it will help you. 
Can you say more about what you need please? We might be able to build a page that lists IDs.

**Comment by o-t-w** [comment]
@captainbrosset I think there are two related use cases:

- What is the ID for a feature?
So to take `Intl.DurationFormat` as an example. Is the ID `intl-durationformat`, `intldurationformat`, `intl-duration-format` etc. Trial and error usually succeeds but can be frustrating.

- Is data for a particular feature even supported at all? 
The feature could be too new. Or it could be included as part of something else. e.g. `Intl.DurationFormat` has its own data, whereas some other methods of `Intl` are included in general support data for `Intl`, rather than having their own ID.

**Comment by captainbrosset** [comment]
Sounds like having a tool for searching the list of feature IDs by BCD keys (roughly equivalent to interface names, methods, events, properties, etc.) would help.

**Comment by captainbrosset** [comment]
@o-t-w would a directory page like this one help: https://web-platform-dx.github.io/web-features-explorer/ids/
I imagine you could use the browser search tool to find things.

**Comment by devnook** [comment]
Feature IDs come from the [github.com/web-platform-dx/web-features](https://github.com/web-platform-dx/web-features/tree/main/features) repo.

Alternatively, you can find the feature ID by browsing the [webstatus.dev](https://webstatus.dev/) site and reading the ID from the URL. E.g. for Intl.ListFormat feature the page URL is https://webstatus.dev/features/intl-list-format, so the ID for this feature is intl-list-format.

I updated the README with this info.

**Comment by devnook** [comment]
Yes, definitely making [https://webstatus.dev/](https://webstatus.dev/) more searchable is a worthwhile thing to do. Thank you for submitting an issue in the webstatus repo!
